config/application.rb

```diff
@@ -21,15 +21,10 @@
 
 module GyakutenCloneGroup
   class Application < Rails::Application
-    # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
-
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration can go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
-
-    # Don't generate system test files.
+    config.i18n.default_locale = :ja
     config.generators.system_tests = nil
+    config.time_zone = 'Tokyo'
+    config.active_record.default_timezone = :local
```

Comment:
ここもタスクとは関係がありませんので，削除しましょう。
```suggestion
    config.generators.system_tests = nil
```